### ISSUE TITLE
Use laminas-navigation instead of custom view helpers for menu and breadcrumbs

### ISSUE THREAD

**Issue by froschdesign** [modir/laminas-book#4]
At the moment, the current page for the menu helper [has to be set manually](https://modir.github.io/laminas-book/users/#adding-view-template-for-login-page), but this can be achieved with the [laminas-navigation](
https://docs.laminas.dev/laminas-navigation/).
There is also no need to write custom view helpers for menu and breadcrumbs as these already exist.

https://modir.github.io/laminas-book/appearance/#writing-own-view-helpers

**Comment by modir** [comment]
> There is also no need to write custom view helpers for menu and breadcrumbs as these already exist.
> https://modir.github.io/laminas-book/appearance/#writing-own-view-helpers

I agree that they are not needed. But I think we should have examples for view helpers. Either we have another example or we write at the top of the article that they are just examples and in reality you would use laminas-navigation for these two use cases.

**Comment by froschdesign** [comment]
> But I think we should have examples for view helpers.

It is correct that the creation of a custom view helper should be shown, but not for navigation. The use of a partial script with a navigation view helper can be explained instead. Example: breadcrumbs with support of [the Schema.org type `BreadcrumbList`](https://schema.org/BreadcrumbList)
And create a custom view helper for the [Bootstrap modal](https://getbootstrap.com/docs/5.3/components/modal/), for example.
